docs/content/configuration/csi_ug_config_create_storageclasses.md

```diff
@@ -24,14 +24,22 @@ _<a name=spaceefficiency>**Table:**</a> `SpaceEfficiency` parameter definitions
 
 - The IBM DS8000 Family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
+- To create a volume with HyperSwap on IBM Spectrum Virtualize storage systems, put a colon (:) between the two pools within the `pool` value. For example:
```

Comment:
@roysahar / @barakda are there any hyperswap prerequisites to perform on the storage side first?

Reply:
@barakda  reminder :) thanks

Reply:
Thanks, @roysahar  I've added additional information here as well as in the compatibility file.
Please view and approve.

Reply:
>For more information about the HyperSwap limitations

why specifically limitations? this page does not mention anything related specifically to the limitations.
I'm not even sure it is necessary to refer to the limitations from this page, especially not before referring to the requirements

Reply:
@oriyarde  I'm assuming you're talking about here: configuration/csi_ug_config_create_storageclasses.md

I've updated the wording a little bit so it's clearer but it's pointing specifically to the limitations page and to the compatibility and requirements pages for more information.
DONE